beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/CommonTestVariables.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (C) 2019-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.beekeeper.integration;
+
+import static java.time.ZoneOffset.UTC;
+
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+
+public final class CommonTestVariables {
```

Comment:
You should probably give this a private constructor to prevent anyone from trying to instantiate it. It's not so important for a test but it's good practice to think about if you need to do something like this in production code.